docs/for_windows_developers.md

```diff
@@ -0,0 +1,113 @@
+# For developers
```

Comment:
That seems a copy-paste of `for_linux_developer.md` file. Keeping two files with such similar content is prone to selective update and lack of synchronisation at some point.

Take a look here:
https://github.com/yaptide/deploy/blob/b363cb84a14af59413e07de9edb5ef7062a663af/docs/simulators.md?plain=1#L63

Such markdown code renders in following way:

<img width="744" alt="Screenshot 2024-03-23 at 23 03 10" src="https://github.com/yaptide/yaptide/assets/7374733/55336d2e-a384-444a-9220-d40492ae84c7">

Maybe you could use similar feature to have just one page with instruction for developers and provide two versions of commands: for Linux and for Windows ?

Reply:
Done